style.css

```diff
@@ -0,0 +1,719 @@
+body {
+	margin: 0;
+	font-family: Montserrat;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)